gui/app.pro

```diff
@@ -28,6 +28,13 @@ unix:OBJECTS_DIR = objects
 unix:RCC_DIR = objects
 
 mac:LIBS += -framework IOKit -framework CoreFoundation
+unix {
```

Comment:
Let's put this under "linux" instead of "unix". I doubt the below will work on FreeBSD/OpenBSD and we do have a few users there.  Then put the body of your new serial_unix.cc into a new file, serial_linux.cc.